locales/en/breaches.ftl

```diff
@@ -2,10 +2,13 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-breach-all-meta-title = { -brand-fx-monitor } - All Data Breaches
+breach-all-meta-page-title = Data Breach Database - { -brand-fx-monitor }
```

Comment:
```suggestion
breach-all-meta-page-title = Data Breach Database — { -brand-fx-monitor }
```

I wonder if you want an em dash in these titles.

Reply:
Also updated `breach-detail-meta-page-title` in [fbb39fd](https://github.com/mozilla/blurts-server/pull/5235/commits/fbb39fd8cbf0815cb0ecc7da52939da31d1fa59e).